File: Modul 6/Soal 4/PRAK504-2210817220007-AISYA ANINDIA SEPTHA.py
def reverse (d) :
    e = 0
    while (int (d)) :
        e = e * 10
        e = e + int (d) % 10
        d = d/10
    return e
a, b = map(int,input().split())
a = reverse (a)
b = reverse (b)
c = a + b
print(reverse (c))

File: Modul 6/Soal 4/PRAK504-2210817220007-AISYA ANINDIA SEPTHA.c
#include <stdio.h>
int reverse(int d){
    int e;
    e = 0;
    while (d != 0){
        e = e * 10;
        e = e + d % 10;
        d = d/10;
        }
    return e;
}
int main() {
    int A, B;
    scanf("%d %d", &A, &B);
    A = reverse (A);
    B = reverse (B);
    int C = A + B;
    printf("%d", reverse (C));
}
